### DISCUSSION TITLE
I've update to version 5 and my graphic design stopped working properly

### DISCUSSION THREAD

**Discussion by AlbertoMPC** [globaleaks/globaleaks-whistleblowing-software#4215] [General]
Finally update 4.11.15 to 5.0.11 , but a purple stripe has appeared on the main page

![web11](https://github.com/user-attachments/assets/bcfaebde-42f3-49c5-9904-9e30e66f51b9)
![web12](https://github.com/user-attachments/assets/1138bb97-c5b3-4a46-aace-f6c739c4dd64)

**Comment by evilaliv3** [comment]
Hello @AlbertoMPC,

This isa actually a known an expected situation.
As we tried to communicate between version 4 and 5 what changes is the client that is rewritten on the base of the up to date Angular (instead of AngularJS) and Bootstrap 5 (instead of Bootstrap 4).
In this update we tried to be fully retrocompatible but of course the new libraries contains some differencies.
For this reason i think that all the visualization problems depends on your CSS and Javascript that would probably need to be updated. If you pass this message to who for you made the customization they will definitely understand how to intervene.
